shared/services/rewards/generator-impl-v8.go

```diff
@@ -852,50 +852,58 @@ func (r *treeGeneratorImpl_v8) checkDutiesForSlot(attestations []beacon.Attestat
 		if inclusionSlot-attestation.SlotIndex > r.beaconConfig.SlotsPerEpoch {
 			continue
 		}
-		rpCommittee, exists := slotInfo.Committees[attestation.CommitteeIndex]
-		if !exists {
-			continue
+		var committeeIndexes []int
+		if attestation.CommitteeBits == nil {
+			committeeIndexes = []int{int(attestation.CommitteeIndex)}
+		} else {
+			committeeIndexes = attestation.CommitteeBits.BitIndices()
```

Comment:
We should probably just write a `func (a AttestationInfo) GetCommitteeIndices() []int` and move this in there so that we don't muddy up this code